ci_scripts/docker-coredeps/deps/build_libetpan.sh

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
I think you should really add #! lines to all these scripts

Reply:
k